terragrunt/org_account/organization/scp.tf

```diff
@@ -87,6 +87,17 @@ data "aws_iam_policy_document" "cds_snc_universal_guardrails" {
       "*"
     ]
   }
+
+  statement {
+    sid = "BlockEC2ForScratch"
+    effect = "Deny"
+    actions = [
+      "ec2:*"
+    ]
```

Comment:
This is too restrictive, it will essentially prevent folks from doing a significant number of things beyond just creating new EC2 Instances, it will prevent networking etc.. 

See this list of what you are blocking and filter it to just the actions we want to prevent: https://docs.aws.amazon.com/service-authorization/latest/reference/list_amazonec2.html